server/lib/Room.js

```diff
@@ -64,7 +87,7 @@ const ROUTER_SCALE_SIZE = config.routerScaleSize || 40;
 class Room extends EventEmitter
 {
 
-	static getLeastLoadedRouter(mediasoupWorkers, peers, mediasoupRouters)
+	static getLeastLoadedRouter(workers, peers, mediasoupRouters)
```

Comment:
You should not use directly `workers`, they are just only for statistics, you must use only `MafaldaRouter` instances. Also, `getLeastLoadedRouter()` is not needed, Mafalda itself manage something similar internally.